x-pack/platform/plugins/shared/ml/public/application/explorer/explorer_data.ts

```diff
@@ -56,7 +58,7 @@ export function getExplorerDefaultState(): ExplorerState {
     indexPattern: getDefaultIndexPattern(),
     influencers: {},
     isAndOperator: false,
-    loading: true,
+    loading: false,
```

Comment:
❓ is it a delbarate change?

Reply:
Yes, we use the `loading` prop to determine whether to show `no jobs selected` view: https://github.com/elastic/kibana/pull/203224/files#diff-ed860f8ee6aaa77969a5132f71414c4da278732ec436bd254ecfe4a4a625c89bL471